rust/core-lib/src/selection.rs

```diff
@@ -16,6 +16,9 @@
 
 use std::cmp::{min, max};
 use std::ops::Deref;
+use std::ops::Bound;
+use std::ops::Bound::*;
```

Comment:
I prefer to avoid glob imports of enums. If you really need to use them a bunch somewhere where they're annoying or cluttery to type out, I'll do something like `use std::ops::Bound as B` or something, in the scope of the particular function where I need it.

I feel this might be kind of weird of me, so 🤷‍♂️. I think it's okay to just use things like `Bound::Included(_)` directly.